WORKFLOW.md

```diff
@@ -61,5 +61,10 @@ This endpoint performs multifaceted duties:
 
 ## Workflow Sequence Diagram:
 
-![Workflow Diagram](./diagrams/Louis%20Login%20Backend%20Sequence%20Diagram.png)
+#### Workflow Sequence UML Document: [Workflow Sequence UML](https://gist.github.com/CFIALeronB/f3e51c1bf166b12bd44155700e4c03b4)
```

Comment:
the four hashmarks seems to be used more as formatting then indicative of headings. don't use headings markers for formatting. I would just remove those and leave the text plain.